Ryujinx.HLE/OsHle/Services/Acc/ProfileBase.cs

```
@@ -0,0 +1,9 @@
+﻿namespace Ryujinx.HLE.OsHle.Services.Acc
+{
+    struct ProfileBase
+    {
+        public byte[] UserId;
```

Imo, instead of the byte array its better to use a struct for the user id with 2 longs inside, with the low and high part of the 128-bits value. Then, expose constructors that allows parsing it from a string or another that allows passing 2 long values. They should throw on any invalid case (string having non 0-9, A-F, a-f character, or length being != 16). This prevents user of the code from creating user ids with invalid lengths.